docusaurus.config.js

```diff
@@ -0,0 +1,143 @@
+const config = {
+  title: 'Nyxx Documentation',
+  tagline: 'A documentation for the Nyxx Discord library',
+  url: url,
+  baseUrl: '/',
+  onBrokenLinks: 'throw',
+  onBrokenMarkdownLinks: 'warn',
+  favicon: 'img/favicon.ico',
+  organizationName: 'nyxx-discord',
+  projectName: 'nyxx',
+
+  presets: [
+    [
+      'classic',
+      /** @type {import('@docusaurus/preset-classic').Options} */
+      ({
+        docs: {
+          sidebarPath: require.resolve('./sidebars.js'),
+
+          editUrl: `https://github.com/${
+            dev ? 'Rapougnac/nyxx_docs' : 'nyxx-discord/nyxx_docs'
+          }/tree/${dev ? 'docusaurus' : 'main'}/`,
+        },
+        blog: {
+          showReadingTime: true,
+          editUrl: `https://github.com/${
+            dev ? 'Rapougnac/nyxx_docs' : 'nyxx-discord/nyxx_docs'
+          }/tree/${dev ? 'docusaurus' : 'main'}/`,
+        },
+        theme: {
+          customCss: require.resolve('./src/css/custom.css'),
+        },
+      }),
+    ],
+  ],
+
+  themeConfig:
+    /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
+    ({
+      metadata: [
+        {
+          name: 'keywords',
+          content: 'discord, bot, bot-framework, nyxx',
+        },
+        {
+          name: 'language',
+          content: 'EN',
+        },
+        {
+          property: 'og:image',
+          content: `${url}img/Nyxx_Logo.png`,
+        },
+      ],
+      navbar: {
+        title: 'Nyxx Docs',
+        logo: {
+          alt: 'Nyxx Logo',
+          src: 'img/Nyxx_Logo.svg',
+        },
+        items: [
+          {
+            type: 'doc',
+            docId: 'intro',
+            position: 'left',
+            label: 'Tutorial',
+          },
+          { to: '/blog', label: 'Blog', position: 'left' },
+          {
+            href: 'https://github.com/nyxx-discord/nyxx',
+            label: 'GitHub',
+            position: 'right',
+          },
+          {
+            type: 'localeDropdown',
+            position: 'right',
+          },
+        ],
+      },
+      footer: {
+        style: 'light',
+        links: [
+          {
+            title: 'Docs',
+            items: [
+              {
+                label: 'Tutorial',
+                to: '/docs/intro',
+              },
+            ],
+          },
+          {
+            title: 'Community',
+            items: [
+              {
+                label: 'Discord',
+                href: 'https://discordapp.com/invite/nyxx',
+              },
+            ],
+          },
+          {
+            title: 'More',
+            items: [
+              {
+                label: 'Blog',
+                to: '/blog',
+              },
+              {
+                label: 'GitHub',
+                href: 'https://github.com/nyxx-discord/nyxx',
+              },
+            ],
+          },
+        ],
+        copyright: `Copyright © ${creationYear} - ${new Date().getFullYear()} Nyxx.`,
```

Comment:
Being nitpicky here, but nyxx is officially spelt with a lowercase n.

Reply:
Yeah, sorry